### PR TITLE
First day project

### PR DESCRIPTION
**Pull request by safar1212**
In this pull request:
I created a react app.
Created components.
Installed linters.

### CODE REVIEW

File: src/components/singleBook.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const SingleBook = () => (
+  <div className="single-book">
+    <div>title</div>
+    <div>author</div>
+  </div>
+);
```
Comment (ShahierNashaat) [inline src/components/singleBook.js]: - [ ] Kindly be noticed that the SingleBook component should have two props the title and the author and renders them.
- [ ] Also it should have a button for removing books the button wouldn't remove the books *yet*,

File: src/components/displayBooks.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import AddBook from './form';
+import SingleBook from './singleBook';
+
+const DisplayBooks = () => (
+  <div className="book-list">
+    <button type="submit">Remove</button>
+    <SingleBook />
+    <AddBook />
+  </div>
+);
```
Comment (ShahierNashaat) [inline src/components/displayBooks.js]: - [ ] Kindly be noticed that you should render a list of books using the `SingleBook` component.
- [ ] Also you should remove the remove button as it should be in the `SingleBook` component.

```
const DisplayBooks = () => 
const books = [];
(
  <div className="book-list">
   books.map((book) => {
       return <SingleBook title={book.title} author={book.author} />
    }
    <AddBook />
  </div>
);
```

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @safar1212 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimemalaika — APPROVED** [review]
Hi @safar1212, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**